### PR TITLE
*: Merge adjacent handles when building key range

### PR DESCRIPTION
**Pull request by shenli**
@coocood @zimulala @tiancaiamao PTAL

### CODE REVIEW

File: executor/executor_pkg_test.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	for _, hr := range hrs {
+		startKey := tablecodec.EncodeRowKeyWithHandle(tid, hr.start)
+		endKey := tablecodec.EncodeRowKeyWithHandle(tid, hr.end)
+		krs = append(krs, kv.KeyRange{StartKey: startKey, EndKey: endKey})
+	}
+	return krs
+}
+
+func (s *testExecSuite) TestMergeHandles(c *C) {
+
```
Comment (zimulala) [inline executor/executor_pkg_test.go]: Remove this empty line.

**Comment by shenli** [comment]
@zimulala PTAL

**Comment by zimulala** [comment]
LGTM

**Comment by shenli** [comment]
@coocood PTAL

**Comment by coocood** [comment]
LGTM

**Pull request merged**